qemu/tests/qemu_disk_img.py

```diff
@@ -53,8 +53,6 @@ def commit(self, drop=False, cache_mode=None, base=None):
         """
         error_context.context("commit snapshot")
         cmds = [self.image_cmd, "commit"]
-        if drop:
```

Comment:
why not remove `drop` from the argument list?

Reply:
Honestly, I even do not know where 'drop' come from. And If you do not add it here, there is no any other options in the 'commit' line.

Reply:
If you remove line `56` and `57`, could you please explain why you still keep `drop` in the argument list of `commit`? Please explain your motivation for this patch in your commit log.

Reply:
From the log info, I can not see the "drop" option. For the commit log, I have updated. Thanks.